2025-02-04 09:35:31,783 INFO    MainThread:580387 [wandb_setup.py:_flush():68] Current SDK version is 0.19.1
2025-02-04 09:35:31,783 INFO    MainThread:580387 [wandb_setup.py:_flush():68] Configure stats pid to 580387
2025-02-04 09:35:31,783 INFO    MainThread:580387 [wandb_setup.py:_flush():68] Loading settings from /home/kliem/.config/wandb/settings
2025-02-04 09:35:31,783 INFO    MainThread:580387 [wandb_setup.py:_flush():68] Loading settings from /home/kliem/FY25/cpyquaticus/Cpyquaticus/cpyquaticus/envs/wandb/settings
2025-02-04 09:35:31,783 INFO    MainThread:580387 [wandb_setup.py:_flush():68] Loading settings from environment variables
2025-02-04 09:35:31,783 INFO    MainThread:580387 [wandb_init.py:_log_setup():528] Logging user logs to /home/kliem/FY25/cpyquaticus/Cpyquaticus/cpyquaticus/envs/wandb/run-20250204_093531-ezflf1zh/logs/debug.log
2025-02-04 09:35:31,783 INFO    MainThread:580387 [wandb_init.py:_log_setup():529] Logging internal logs to /home/kliem/FY25/cpyquaticus/Cpyquaticus/cpyquaticus/envs/wandb/run-20250204_093531-ezflf1zh/logs/debug-internal.log
2025-02-04 09:35:31,783 INFO    MainThread:580387 [wandb_init.py:init():644] calling init triggers
2025-02-04 09:35:31,783 INFO    MainThread:580387 [wandb_init.py:init():650] wandb.init called with sweep_config: {}
config: {'exp_name': 'petting_zoo_train', 'seed': 1, 'torch_deterministic': True, 'cuda': True, 'track': True, 'wandb_project_name': 'cleanRL', 'wandb_entity': None, 'capture_video': False, 'env_id': 'CartPole-v0', 'total_timesteps': 100000, 'learning_rate': 0.00025, 'num_envs': 4, 'num_steps': 128, 'anneal_lr': True, 'gamma': 0.99, 'gae_lambda': 0.95, 'num_minibatches': 4, 'update_epochs': 4, 'norm_adv': True, 'clip_coef': 0.2, 'clip_vloss': True, 'ent_coef': 0.01, 'vf_coef': 0.5, 'max_grad_norm': 0.5, 'target_kl': None, 'batch_size': 512, 'minibatch_size': 128, 'num_iterations': 195}
2025-02-04 09:35:31,783 INFO    MainThread:580387 [wandb_init.py:init():680] starting backend
2025-02-04 09:35:31,783 INFO    MainThread:580387 [wandb_init.py:init():684] sending inform_init request
2025-02-04 09:35:31,787 INFO    MainThread:580387 [backend.py:_multiprocessing_setup():104] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2025-02-04 09:35:31,788 INFO    MainThread:580387 [wandb_init.py:init():697] backend started and connected
2025-02-04 09:35:31,789 INFO    MainThread:580387 [wandb_init.py:init():790] updated telemetry
2025-02-04 09:35:31,794 INFO    MainThread:580387 [wandb_init.py:init():822] communicating run to backend with 90.0 second timeout
2025-02-04 09:35:31,797 WARNING MsgRouterThr:580387 [router.py:message_loop():75] message_loop has been closed
2025-02-04 09:35:41,797 ERROR   MainThread:580387 [wandb_init.py:init():1308] error in wandb.init()
Traceback (most recent call last):
  File "/home/kliem/FY25/cpyquaticus/cpyq/lib/python3.10/site-packages/wandb/sdk/wandb_init.py", line 1298, in init
    return wi.init()
  File "/home/kliem/FY25/cpyquaticus/cpyq/lib/python3.10/site-packages/wandb/sdk/wandb_init.py", line 825, in init
    result = run_init_handle.wait(
  File "/home/kliem/FY25/cpyquaticus/cpyq/lib/python3.10/site-packages/wandb/sdk/lib/mailbox.py", line 277, in wait
    raise MailboxError("transport failed")
wandb.sdk.lib.mailbox.MailboxError: transport failed
